dask/tests/test_threaded.py

```diff
@@ -147,7 +144,7 @@ def test_interrupt():
             proc.send_signal(sigint)
             # Wait a bit for it to die
             start = time()
-            while time() - start < 2.0:
+            while time() - start < 4.0:
```

Comment:
Yeah, I feel your pain on this.  Dask/distributed has a few of these going on right now.